h2o-admissibleml/src/main/java/hex/Infogram/InfogramUtils.java

```diff
@@ -178,13 +196,30 @@ public static Frame extractTrainingFrame(InfogramParameters parms, String[] sens
     return modelBuilders;
   }
   
+  public static Frame generateCMIRelevance(String[] allPredictorNames, double[] admissible, double[] admissibleIndex, 
+                                           double[] relevance, double[] cmi, double[] cmiRaw, boolean buildCore) {      
+    Vec.VectorGroup vg = Vec.VectorGroup.VG_LEN1;
+    Vec vName = Vec.makeVec(allPredictorNames, vg.addVec());
+    Vec vAdm = Vec.makeVec(admissible, vg.addVec());
+    Vec vAdmIndex = Vec.makeVec(admissibleIndex, vg.addVec());
+    Vec vRel = Vec.makeVec(relevance, vg.addVec());
+    Vec vCMI = Vec.makeVec(cmi, vg.addVec());
+    Vec vCMIRaw = Vec.makeVec(cmiRaw, vg.addVec());
+    String[] columnNames = buildCore ? new String[]{"column", "admissible", "admissible_index", "total_information",
+            "net_information", "net_information_raw"} : new String[]{"column", "admissible", "admissible_index",
+            "relevance_index", "safety_index", "safety_index_raw"};
+    Frame cmiRelFrame = new Frame(Key.<Frame>make(), columnNames, new Vec[]{vName, vAdm, vAdmIndex, vRel, vCMI, vCMIRaw});
+    DKV.put(cmiRelFrame);
+    return cmiRelFrame;
+  }
+  
   public static void removeFromDKV(List<Key<Frame>> generatedFrameKeys) {
     for (Key<Frame> oneFrameKey : generatedFrameKeys)
         DKV.remove(oneFrameKey);
   }
-
+  
   /***
-   * To calculate the CMI, refer to https://h2oai.atlassian.net/browse/PUBDEV-8075 section I step 2 for core infogram,
+   * To calculate the cim, refer to https://h2oai.atlassian.net/browse/PUBDEV-8075 section I step 2 for core infogram,
```

Comment:
typo

Reply:
@koniecsveta CMI = Conditional Mutual Information.  It is not a typo.

By the way, what is cim?

Reply:
yesyes, but here`CMI` was changed to `cim`, so it should change back to `CIM`. I don't know what is `cim`.

Reply:
It is conditional mutual information, CMI not CIM.

Reply:
@wendycwong, if you look into this comment on the main PR page, it looks that @koniecsveta suggested change `CMI  `to `cim`, but it is not true. For some reason, you changed `CMI ` to  `cim`, that is what @koniecsveta means by her comment. :) Look into FIles changes. ;) But it is just a detail. :)

```suggestion
   * To calculate the CMI, refer to https://h2oai.atlassian.net/browse/PUBDEV-8075 section I step 2 for core infogram,
```